lib/publish.js

```diff
@@ -26,43 +25,45 @@ module.exports = async (pluginConfig, {options: {branch, repositoryUrl}, nextRel
   if (assets && assets.length > 0) {
     const globbedAssets = await globAssets(assets);
     debug('globed assets: %o', globbedAssets);
-    // Make requests serially to avoid hitting the rate limit (https://developer.github.com/v3/guides/best-practices-for-integrators/#dealing-with-abuse-rate-limits)
-    await pReduce(globbedAssets, async (_, asset) => {
-      const filePath = isPlainObject(asset) ? asset.path : asset;
-      let file;
 
-      try {
-        file = await stat(filePath);
-      } catch (err) {
-        logger.error('The asset %s cannot be read, and will be ignored.', filePath);
-        return;
-      }
-      if (!file || !file.isFile()) {
-        logger.error('The asset %s is not a file, and will be ignored.', filePath);
-        return;
-      }
+    await Promise.all(
+      globbedAssets.map(async asset => {
+        const filePath = isPlainObject(asset) ? asset.path : asset;
+        let file;
 
-      const fileName = asset.name || basename(filePath);
-      const upload = {
-        owner,
-        repo,
-        url: uploadUrl,
-        file: await readFile(filePath),
-        contentType: mime.getType(extname(fileName)) || 'text/plain',
-        contentLength: file.size,
-        name: fileName,
-      };
+        try {
+          file = await stat(filePath);
+        } catch (err) {
+          logger.error('The asset %s cannot be read, and will be ignored.', filePath);
+          return;
+        }
+        if (!file || !file.isFile()) {
+          logger.error('The asset %s is not a file, and will be ignored.', filePath);
+          return;
+        }
 
-      debug('file path: %o', filePath);
-      debug('file name: %o', fileName);
+        const fileName = asset.name || basename(filePath);
+        const upload = {
+          owner,
+          repo,
```

Comment:
`uploadAsset()` has no `owner` and `repo` options: https://octokit.github.io/rest.js/#api-Repos-uploadAsset

Reply:
It's unrelated to this PR (it was there before), so I added this change in a different commit.